packages/broker/src/plugins/operator/OperatorPlugin.ts

```diff
@@ -52,12 +50,13 @@ export class OperatorPlugin extends Plugin<OperatorPluginConfig> {
     constructor(options: PluginOptions) {
         super(options)
         const provider = new JsonRpcProvider(this.brokerConfig.client.contracts!.streamRegistryChainRPCs!.rpcs[0].url)
+        // TODO read from client, as we need to use production value in production environment (not random address)
```

Comment:
I think the constructor maybe should take a Wallet? At least in browser it makes sense since we don't have the private key (signer is metamask or similar)

Reply:
then you would pass that Wallet around, and things like RPC URLs would be Someone Else's Problem(TM).

Reply:
There is a separate ticket to implement valid authentication/signing to operator plugin: https://linear.app/streamr/issue/NET-1034/expose-signer-from-streamclient